http_connector.py

```diff
@@ -1,848 +1,848 @@
-            ret_val = self._handle_http_get(param)
-
-        elif action_id == 'http_post':
-            ret_val = self._handle_http_post(param)
-
-        elif action_id == 'http_put':
-            ret_val = self._handle_http_put(param)
-
-        elif action_id == 'http_patch':
-            ret_val = self._handle_http_patch(param)
-
-        elif action_id == 'http_delete':
-            ret_val = self._handle_http_delete(param)
-
-        elif action_id == 'http_head':
-            ret_val = self._handle_http_head(param)
-
-        elif action_id == 'http_options':
-            ret_val = self._handle_http_options(param)
-
-        elif action_id == 'get_file':
-            ret_val = self._handle_get_file(param, 'get')
-
-        elif action_id == 'put_file':
-            ret_val = self._handle_put_file(param, 'post')
-
-        return ret_val
-
-
-if __name__ == '__main__':
-
-    import argparse
-
-    import pudb
-    pudb.set_trace()
-
-    argparser = argparse.ArgumentParser()
-
-    argparser.add_argument('input_test_json', help='Input Test JSON file')
-    argparser.add_argument('-u', '--username', help='username', required=False)
-    argparser.add_argument('-p', '--password', help='password', required=False)
-    argparser.add_argument('-v', '--verify', action='store_true', help='verify', required=False, default=False)
-
-    args = argparser.parse_args()
-    verify = args.verify
-    session_id = None
-    verify = args.verify
-
-    if args.username and args.password:
-        login_url = BaseConnector._get_phantom_base_url() + "login"
-        try:
-            print("Accessing the Login page")
-            r = requests.get(login_url, verify=verify, timeout=DEFAULT_REQUEST_TIMEOUT)
-            csrftoken = r.cookies['csrftoken']
-            data = {'username': args.username, 'password': args.password, 'csrfmiddlewaretoken': csrftoken}
-            headers = {'Cookie': 'csrftoken={0}'.format(csrftoken), 'Referer': login_url}
-
-            print("Logging into Platform to get the session id")
-            r2 = requests.post(login_url, verify=verify, data=data, headers=headers, timeout=DEFAULT_REQUEST_TIMEOUT)
-            session_id = r2.cookies['sessionid']
-
-        except Exception as e:
-            print("Unable to get session id from the platform. Error: {0}".format(str(e)))
-            sys.exit(1)
-
-    if len(sys.argv) < 2:
-        print("No test json specified as input")
-        sys.exit(0)
-
-    with open(sys.argv[1]) as f:
-        in_json = f.read()
-        in_json = json.loads(in_json)
-        print(json.dumps(in_json, indent=4))
-
-        connector = HttpConnector()
-        connector.print_progress_message = True
-
-        if session_id is not None:
-            in_json['user_session_token'] = session_id
-
-        ret_val = connector._handle_action(json.dumps(in_json), None)
-        print(json.dumps(json.loads(ret_val), indent=4))
-
-    sys.exit(0)
+# File: http_connector.py

```

Comment:
Same here. Can only changes be marked in the diff?

Reply:
Done.